profile-widget/qmlprofile.cpp

```diff
@@ -23,6 +27,9 @@ QMLProfile::QMLProfile(QQuickItem *parent) :
 
 void QMLProfile::paint(QPainter *painter)
 {
+	QElapsedTimer timer;
+	timer.start();
```

Comment:
Since the result is only read if `verbose == true`, perhaps also put `timer.start()` in a `if (verbose)`?

Reply:
I thought about it but was too lazy. Will rectify that tardiness.

Reply:
fixed